requirements.txt

```diff
@@ -7,3 +7,4 @@ pandas
 scikit-learn>=0.22
 scipy>=1.3.3
 threadpoolctl
+bokeh
```

Comment:
I hit an error on my first run, as my version of bokeh was apparently outdated, should this be set to specific version num? Updating from 1.3.4 (?!) to 2.0.1 fixed it. 
For completion sake, error:
```
Traceback (most recent call last):
  File "/Users/logan/opt/anaconda3/bin/tedana", line 11, in <module>
    load_entry_point('tedana', 'console_scripts', 'tedana')()
  File "/Users/logan/local_code/tedana/tedana/workflows/tedana.py", line 671, in _main
    tedana_workflow(**kwargs)
  File "/Users/logan/local_code/tedana/tedana/workflows/tedana.py", line 616, in tedana_workflow
    reporting.generate_report(out_dir=out_dir, tr=img_t_r)
  File "/Users/logan/local_code/tedana/tedana/reporting/html_report.py", line 82, in generate_report
    kappa_rho_plot = df._create_kr_plt(comptable_cds)
  File "/Users/logan/local_code/tedana/tedana/reporting/dynamic_figures.py", line 151, in _create_kr_plt
    legend_group='classif')
  File "fakesource", line 5, in circle
  File "/Users/logan/opt/anaconda3/lib/python3.7/site-packages/bokeh/plotting/helpers.py", line 857, in func
    glyph = _make_glyph(glyphclass, kwargs, glyph_ca)
  File "/Users/logan/opt/anaconda3/lib/python3.7/site-packages/bokeh/plotting/helpers.py", line 398, in _make_glyph
    return glyphclass(**kws)
  File "/Users/logan/opt/anaconda3/lib/python3.7/site-packages/bokeh/model.py", line 307, in __init__
    super(Model, self).__init__(**kwargs)
  File "/Users/logan/opt/anaconda3/lib/python3.7/site-packages/bokeh/core/has_props.py", line 253, in __init__
    setattr(self, name, value)
  File "/Users/logan/opt/anaconda3/lib/python3.7/site-packages/bokeh/core/has_props.py", line 288, in __setattr__
    (name, self.__class__.__name__, text, nice_join(matches)))
AttributeError: unexpected attribute 'legend_group' to Circle, possible attributes are angle, angle_units, fill_alpha, fill_color, js_event_callbacks, js_property_callbacks, line_alpha, line_cap, line_color, line_dash, line_dash_offset, line_join, line_width, name, radius, radius_dimension, radius_units, size, subscribed_events, tags, x or y
```

Reply:
I think version 1.4 is the minimum required but I'm not completely sure right now. Let me check.

Reply:
Strange, on OS X I got a working version. Would be good to set a minimum version in the `.txt` file though.

Reply:
I was using an anaconda install, from not so long ago (~months). As you can see, the error message is super useless - so I just had to assume it was a version issue.

Reply:
Interestingly, if we require Bokeh 2.0.0 then we have to drop python 3.5 support (since it requires python 3.6+). How would you feel about that @tsalo  ?